src/utils/PositionContainer/PositionContainer.component.tsx

```diff
@@ -147,12 +147,15 @@ export const PositionContainer = forwardRef(
             shadow={shadow}
             width={width}
         >
-            <Background background={background} />
+            <Background className="position-bg" background={background} />
```

Comment:
A minor consideration, but most class names for anchor components are prefaced with `anchor-`.